scripts/eval_harness.py

```diff
@@ -0,0 +1,147 @@
+import argparse
+import os
+
+import lm_eval
+import torch
+import torch._inductor.config
+from lm_eval.utils import make_table
+from torch import distributed as dist
+
+from fms.models import get_model
+from fms.utils import evaluation, tokenizers
+
+
+"""
+Example use:
+```
+$ srun -N 1 --gres=gpu:1 --cpus-per-task=12 --mem=128G --unbuffered --gres-flags=enforce-binding  python scripts/eval_harness.py --model_path=~/models/7B-F/ --model_source=meta  --tokenizer=~/models/tokenizer.model --tasks=hellaswag
+
+|  Tasks  |Version|Filter|n-shot| Metric |Value |   |Stderr|
+|---------|-------|------|-----:|--------|-----:|---|-----:|
+|hellaswag|Yaml   |none  |     0|acc     |0.5739|±  |0.0049|
+|         |       |none  |     0|acc_norm|0.7349|±  |0.0044|
```

Comment:
I see a higher accuracy (77.2) here: https://paperswithcode.com/sota/sentence-completion-on-hellaswag

not sure if this is an issue with how I'm using the lm eval harness, or with our implementation of llama, or some other anomaly.

Reply:
using num_shots=10 (and the fine tuned model) fixes this